meetings/2023-05-24.md

```diff
@@ -0,0 +1,60 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2023-05-24
+
+## Links
+
+* **Recording**:  <https://www.youtube.com/watch?v=gWiRF-Nozjw>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1391>
+* **Minutes Google Doc**: <https://docs.google.com/document/d/1b31yXNoPnPkSdwEuKQoeGUVYbGKjlSff1bi9XBYslBY/edit>
+
+## Present
+
+* Chengzhong Wu @legendecas (voting member)
+* Rafael Gonzaga @RafaelGSS (voting member)
+* Richard Lau @richardlau (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Matteo Collina @mcollina (voting member)
+* Gireesh Punathil @gireeshpunathil (voting member)
+
+## Agenda
+
+### Announcements
+
+* 2023 TSC Elections  [#1382](https://github.com/nodejs/TSC/issues/1382)
+  * Michael Dawson is the only candidate for Chair
+  * Matteo Collina is the only candidate for vice-chair
+  * Congratulations to both!
+
+### CPC and Board Meeting Updates
+
+<https://github.com/openjs-foundation/cross-project-council/pull/1055> - proposed changes to the Foundation bylaws.
+
+\*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/TSC
+
+* Moderation Team annual certification [#1390](https://github.com/nodejs/TSC/issues/1390)
+  * Done
+
+* Adding the OpenJS Foundation staff as a billing manager for nodejs-private [#1376](https://github.com/nodejs/TSC/issues/1376)
+  * Rafael: we can disable the Mac and Windows
+  * Matteo: I’ve disabled the Mac and Windows workflow in nodejs-private
+  * Moshe: we can trigger the Mac Workflow only when a specific label is present
+  * Richard: we might disable macOS altogether from GitHub Actions because it’s very costly even for GitHub.
+  * Richard: depending on how much activity there is in the private repo, we might still run out of minutes.
+  * Matteo: take the action of getting a billing manager from the Foundation and add a small budget to it (100$/month) and see how it plays out.
+
+* Opportunity: Grace Hopper Open Source Day (virtual) Sept 22, 2023 [#1372](https://github.com/nodejs/TSC/issues/1372)
+  * Rafael is planning to create the private curated list any time soon. Likely next week.
```

Comment:
Is this supposed to say "not create...any time soon"? Or "create...some time soon?"

Reply:
I guess we could just remove it and leave "likely next week":

```suggestion
  * Rafael is planning to create the private curated list likely next week.
```